bika/lims/locales/es/LC_MESSAGES/bika.po

```diff
@@ -5306,59 +5775,83 @@ msgid "Select the currency the site will use to display prices."
 msgstr "Seleccione la moneda que se debe utilizar al mostrar los importes."
 
 #: bika/lims/content/analysisservice.py:230
-msgid "Select the default container to be used for this analysis service. If the container to be used depends on the sample type and preservation combination, specify the container in the sample type table below"
-msgstr "Seleccionar el recipiente por defecto que debe ser utilizado para este servicio de análisis. Si el recipiente depende de la combinación del tipo de muestra y del método de conservación, deberá especificar el recipiente específico para cada tipo de de muestra."
+msgid ""
+"Select the default container to be used for this analysis service. If the "
+"container to be used depends on the sample type and preservation "
+"combination, specify the container in the sample type table below"
+msgstr ""
+"Seleccionar el recipiente por defecto que debe ser utilizado para este "
+"servicio de análisis. Si el recipiente depende de la combinación del tipo de "
+"muestra y del método de conservación, deberá especificar el recipiente "
```

Comment:
@xispa I wouldn't say _especificar el recipiente especifíco para_. Maybe _especificar el recipiente para_ is enough?